languageserver/server/server.go

```diff
@@ -3116,7 +3137,22 @@ func convertHint(
 				},
 			}
 		}
+		protocolDiagnostic = protocol.Diagnostic{
+			Message: linterDiagnostic.Message,
+			// protocol.SeverityHint doesn't look prominent enough in VS Code,
+			// only the first character of the range is highlighted.
+			Severity: protocol.SeverityInformation,
+			Range:    protocolRange,
+		}
```

Comment:
Duplicated here and below. Maybe refactor out the assignment to `protocolDiagnostic` after the switch. The only difference in the cases is the message